pouchdb-core/index.d.ts

```diff
@@ -426,6 +431,10 @@ declare namespace PouchDB {
         on(event: 'created', listener: (dbName: string) => any): this;
         on(event: 'destroyed', listener: (dbName: string) => any): this;
 
+        debug: debug.IDebug;
+
+        adapter(name: string, adapter: any): void;
```

Comment:
This is for adding a new pluggable adapter.  Agreed the main pouch site doesn't mention it, I got the spec from the worker-pouch plugin. https://github.com/nolanlawson/worker-pouch